content/_changelogs/10.8.0.md

```diff
@@ -0,0 +1,106 @@
+## 10.7.0
```

Comment:
```suggestion
## 10.8.0
```

Reply:
OMG....I had 10.9.0 before pushing. I must not want to release as 10.8.0 😅